### DISCUSSION TITLE
Hydration error

### DISCUSSION THREAD

**Discussion by AnH13-6** [neuefische/questions#197] [Web]
Hey, I get a hydration error message after adding a form with the POST method for my api to my app. I somewhat understand what the problem is after reading up on it, but I still don't know how to fix this. There are no errors in my actual code. 
Here the error message: 
<img width="960" alt="Screenshot 2022-09-30 at 11 20 16" src="https://user-images.githubusercontent.com/60193027/193246961-edc67d6e-08ae-415c-b146-e8a9aad12860.png">

And here's my code: 
https://github.com/AnH13-6/capstone-project-two/tree/book-entry-form

Thanks in advance!

**Comment by actyralf** [comment]
I cannot reproduce this error with the branch you posted. Instead there seems to be a fetching problem... Are you sure you pushed your code and posted the correct branch?
